### ISSUE TITLE
Conda can not parse environment.yml file

### ISSUE THREAD

**Issue by alexandreroutier** [aramis-lab/clinica#15]
Hi!

I had some feedback from a Clinica user: when using the latest version of `conda` using Miniconda, the YAML file can not be parsed. 

I think I managed to reproduce this issue. It seems that this due to the migration of conda 4.6 to  conda 4.7:
```
$ conda env create -f environment.yml --name ClinicaSandBox
Collecting package metadata: done
Solving environment: done


==> WARNING: A newer version of conda exists. <==
  current version: 4.6.14
  latest version: 4.7.11

Please update conda by running

$ conda update -n base -c defaults conda
[...]
$ conda update -n base -c defaults conda
[...]
$ conda --version
conda 4.7.11
$ conda env create -f environment.yml --name ClinicaSandBox2
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound: 
  - python==3.6
```

To solve this issue, you simply need to replace `python==3.6` by  `python=3.6` in `environment.yml`.

I don't know if there is an alternative bugfix for older conda installations.

Alexandre

**Comment by mdiazmel** [comment]
Hi, 
I tested the bugfix with conda 4.0.5 (the first version stamped >4.0) and it works. 
I'll propose a PR to fix this bug.

**Comment by mdiazmel** [comment]
Issue solved by PR #43 .
